zebra-utils/src/bin/zebra-checkpoints/main.rs

```diff
@@ -227,15 +234,19 @@ async fn main() -> Result<()> {
 
                 let block_bytes: Vec<u8> = hex::decode(block_bytes)?;
 
-                // TODO: is it faster to call both `getblock height 0` and `getblock height 1`,
-                //       rather than deserializing the block and calculating its hash?
+                // TODO: is it faster to call both `getblock height version=0`
+                //       and `getblock height version=1`, rather than deserializing the block
```

Comment:
I spent a while trying to figure out what's so special about heights 0 and 1 that we mention them here. Then I finally noticed you added the versions!

Reply:
Yeah it was a confusing way of saying it, because "height" was meant to stand for an actual height, then the second parameter is the verbosity of the RPC. (And it's optional.)